src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -346,6 +347,40 @@ const CallTable = ({
             },
             ...(isAzure // GCP workspaces do not have inputs/outputs/logs (only log file is at top level, linked in workflow dashboard)
               ? [
+                  {
+                    size: { basis: 200, grow: 1 },
+                    field: 'cost',
+                    headerRenderer: () =>
+                      h(Sortable, { sort, field: 'cost', onSort: setSort }, [
+                        'Approximate Task Cost',
+                        h(
+                          TooltipTrigger,
+                          {
+                            content:
+                              'This is a calculated list price for VM costs per hour and does not include disk cost, subworkflow costs, or any cloud account discounts.',
+                          },
+                          [icon('info-circle', { style: { marginLeft: '0.4rem', color: colors.accent(1) } })]
+                        ),
+                      ]),
+                    cellRenderer: ({ rowIndex }) => {
+                      const { vmCostUsd, taskStartTime, taskEndTime, executionStatus, callCaching } = filteredCallObjects[rowIndex];
+                      if (taskStartTime && vmCostUsd) {
+                        if (executionStatus === 'Running') {
+                          const cost = getTaskCost(vmCostUsd, taskStartTime);
+                          return div({}, [span({ style: { fontStyle: 'italic' } }, ['In Progress - ']), `$${cost}`]);
```

Comment:
Nit: empty props objects can be omitted.

```suggestion
                          return div([span({ style: { fontStyle: 'italic' } }, ['In Progress - ']), `$${cost}`]);
```